media/src/main/java/io/wcm/handler/media/impl/MediaFormatResolver.java

```diff
@@ -209,6 +207,7 @@ private void generateMediaFormatsForWidths(@NotNull Map<String, MediaFormatOptio
           .extensions(mediaFormat.getExtensions())
           .ratio(mediaFormat.getRatio())
           .width(widthOption.getWidth())
+          .property("parentMediaFormat", setParent ? mediaFormat : null)
```

Comment:
we should create a constant for this string, probably in `MediaNameConstants` with prefix `MEDIAFORMAT_PROP_`

Reply:
done